website/cue/reference/components.cue

```diff
@@ -439,11 +443,26 @@ components: {
 		default_namespace: string
 	}
 
-	#Output: {
+	#OutputData: {
```

Comment:
It looks like this somehow ends up rendering has the hash anchor. There are some links that look for `#output` currently that will need to be updated to `#output-data` (see the failing Netlify build).

Would it make sense to have `LogOutput` and `MetricOutput` have an additional, optional field, defaulting to the default output, for the output they are from? We could also leave this until we have an actual example output from a named output.

Reply:
I'm alright with leaving the additional field on `LogOutput` and `MetricOutput` until there is a more concrete use case, since it seems like it would always currently default to the default output. Perhaps it would make more sense to make `OutputData` a part of each `Output` eventually?

Reply:
👍 I'm fine with just deferring it until we have an example. We could drop `data?` below since it isn't currently used.

Reply:
Sounds good. I was considering populating `data` for sources and relevant transforms with their current `output` field but decided to hold back, so `data` isn't currently used at the moment. In theory, with `data` filled out, we'd have brought `outputs` and `output` together. We could then remove the `Output Data` section and fix up the UI so each output also shows its output data. But, that'd bring in a lot more changes, and I figure can be left for a follow up if we go that route.

Reply:
👍 yeah, I'm down to see that is a future refactoring.